docs/5.0/pages/enterprise/quickstart-enterprise.md

```diff
@@ -119,27 +118,64 @@ ssh_service:
   enabled: "yes"
 ```
 
+## Start Teleport on _auth.example.com_
 
-### Systemd Unit File
-
-Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/master/examples/systemd)
-on Github and save it as `/etc/systemd/system/teleport.service` on both servers.
+Start the Teleport service and confirm the service has started. After it has started you can get the Certificate Authority (CA) pin that has been generated.
 
 ```bsh
-# run this on both servers:
-$ sudo systemctl daemon-reload
-$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+# confirm Teleport started
+$ sudo systemctl status teleport
+# after successful start retrieve the CA pin for using in the node
+$ sudo tctl status
+Cluster  teleport
+Version  {{ version }}
+Host CA  never updated
+User CA  never updated
+Jwt CA   never updated
+CA pin   sha256:58b1c5cb8885d84ce08f3e3a47acb33d49841f12dd7648c0c1dd31bd501628
+```
+
+Save the following configuration file as `/etc/teleport.yaml` on the _node.example.com_:
+
+!!! note "Note"
+
+    Make sure to replace the ca_pin with the above `tctl status` output and the right auth server address.
+
+```yaml
+teleport:
+  nodename: node.example.com
+  auth_token: dogs-are-much-nicer-than-cats
+  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  auth_servers: [ "auth.example.com:3025" ]
+  # Set the ca_pin value so it can verify the Teleport server
+  ca_pin: sha256:58b1c5cb8885d84ce08f3e...
+
+  # enable ssh service and disable auth and proxy:
+ssh_service:
+  enabled: true
+auth_service:
+  enabled: false
+proxy_service:
+  enabled: false
 ```
 
-## Starting
+
+## Start Node service
 
 ```bsh
-# run this on both servers:
+# run this on node server:
 $ sudo systemctl start teleport
+# Confirm node has started successfully and connected
+$ sudo systemctl status teleport
 ```
+## Teleport Status and Networking
+
+You can review the logs of the Teleport service with `journalctl -fu teleport` and see its current status
+with `sudo systemctl status teleport`.
 
-Teleport daemon should start and you can use `netstat -lptne` to make sure that
-it's listening on [TCP/IP ports](../admin-guide.md#ports). On _auth.example.com_, it should
+You can use `netstat -lptne` to review the port that Teleport is
```

Comment:
Mixed use of sudo and no sudo on these commands. Would it be a good idea to include sudo?